### PR TITLE
[Feature] [EDT-1472] Add options to control start/stop/end characters

### PR DESCRIPTION
**Pull request by NicolaVerbeeck**
This PR

## PR Guidelines

- [X] I have read the [contribution and PR guidelines](/chili-publish/studio-sdk/blob/main/CONTRIBUTING.md)

## Related tickets

-   [EDT-1472](https://chilipublishintranet.atlassian.net/browse/EDT-1472)

## Screenshots

[EDT-1472]: https://chilipublishintranet.atlassian.net/browse/EDT-1472?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Dvergar — COMMENTED** [review]
We're sending `drawStartStopChars` to engine but it seems that we do not support it yet? I saw that the `dart_barcode` lib merged the related PR but what about engine?

**Comment by NicolaVerbeeck** [comment]
Engine is ready but waiting for release on the dart barcode lib before creating PR

File: packages/sdk/src/controllers/BarcodeController.ts [hunk 1]
```diff
@@ -233,7 +259,12 @@ export class BarcodeController {
                 quietZoneAlwaysCombined = true;
                 break;
             case BarcodeType.ean13:
+                allowToggleText = false;
+                allowToggleDrawEndChar = true;
+                break;
             case BarcodeType.ean8:
+                allowToggleDrawStartAndEndChar = true;
```
Comment (brapoprod) [inline packages/sdk/src/controllers/BarcodeController.ts:266]:
```suggestion
                allowToggleDrawStartAndStopChar = true;
```

**Review by brapoprod — CHANGES_REQUESTED** [review]
Jeroen asks to align the SDK methods with the copy that is used:
allowLightMarginToggle
allowStartStopCharactersToggle

Also the setters should be renamed accordingly

**Comment by NicolaVerbeeck** [comment]
> Jeroen asks to align the SDK methods with the copy that is used: allowLightMarginToggle allowStartStopCharactersToggle
> 
> Also the setters should be renamed accordingly

That means that for every terminology for every barcode type we need to add more booleans :/

**Pull request closed without merging**